### DISCUSSION TITLE
1-m-Relation with a JSON field

### DISCUSSION THREAD

**Discussion by robkuz** [prisma/prisma#7288] [Q&A] [answered]
Hi,
is it possible to model a 1-m-relationship with a JSON field?
Lets say I have this schema

```prisma
model User {
  id      Int      @id @default(autoincrement())
  email   String   @unique
  name    String?
  posts   Post[]
}

model Post {
  createdAt  DateTime 
  title      String
  published  Boolean 
}
```

now assume that `User.posts` is a json database field and we now for sure that field will always contain an array of `Post` instances, Is it somehow possible to advise Prisma to automatically de-/serialize this field when reading and writing from the DB?

instead of accessing it later like this
```
const posts = user.posts as Prisma.JsonObject
```

**Comment by ryands17** [comment, marked as answer]
@robkuz 👋 

This is currently not possible. Also the `Json` field would be in the same table so I would like to understand your use case for the same. Relations are currently only possible with a foreign key on other tables.
